frontend/src/config.ts

```diff
@@ -63,12 +67,36 @@ function getAnnouncementActive() {
 
 // Feature Launch Announcements
 // If `isActive` is false, the modal will not proc for ANY user
-export const ANNOUNCEMENT = {
+export const ANNOUNCEMENT: any = {
   isActive: getAnnouncementActive(),
-  version: t('announcement.version')``,
   title: t('announcement.title')``,
   subtext: t('announcement.subtext')``,
-  imageUrl: t('announcement.imageUrl')``,
-  buttonUrl: t('announcement.buttonUrl')``,
-  buttonText: t('announcement.buttonText')``,
+  mediaUrl: t('announcement.mediaUrl')``,
+  primaryButtonUrl: t('announcement.primaryButtonUrl')``,
+  primaryButtonText: t('announcement.primaryButtonText')``,
+  secondaryButtonUrl: t('announcement.secondaryButtonUrl')``,
+  secondaryButtonText: t('announcement.secondaryButtonText')``,
+}
+
+// Lazily compute the announcement version and memoize it for future use.
+let memoizedVersion: string | null = null
+export async function getAnnouncementVersion(): Promise<string> {
```

Comment:
Minor nitpick, since we are already exporting `getAnnouncementVersion`, perhaps we can use that elsewhere instead. It might not be obvious that you have to call `await` on `ANNOUNCEMENT.version`.